controllers/AuthenticationController.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
le code de the game ne sera  pas applicable ici et surtout tu n'as pas de signup, juste un login. donc pour l'instant laisse le vide.